docker/debian/Dockerfile

```diff
@@ -1,13 +1,20 @@
 FROM debian:buster
 MAINTAINER Rob Gil (rob@rem5.com)
+
 ENV DEBIAN_FRONTEND noninteractive
 ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
-RUN apt-get update
-RUN apt-get install -y libpcre3-dev apt-transport-https ca-certificates curl wget logrotate \
-    libc-ares2 libjson-c3 vim systemd procps libreadline7 gnupg2 lsb-release apt-utils
+
+RUN apt-get update && \
```

Comment:
I think you can further merge the RUN commands. This one can be merged with next two RUN commands.

Reply:
Hi,
Indeed, but maybe it would be easier to merge the two together (curl + echo).
Indeed, if we join the three together, then we always have to rebuild even if the key changes.

Granted, it won't happen everyday, so as you wish ;-)

Reply:
Just some thoughts, If you add the keys and the repository before `apt-get update` in line 7 you can save some build time cause you don't have to run `apt-update` a second time in line 15. You can install then `frr` and `frr-pythontools` altogether and you also don't have to run `rm -rf /var/lib/apt/lists/*` a second time.

Reply:
Its not a big change/improvement either way. I am good with either. Its your call.

Reply:
This is the recommended way doing package handlings to avoid stale caches: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/